src/experimental-features.js

```diff
@@ -0,0 +1,26 @@
+export class ExperimentalFeatures {
```

Comment:
maybe can move to services folder, i don't know what you think

Reply:
Yes and also convert this into typescript class would be ok too.